Cargo.toml

```diff
@@ -30,6 +30,7 @@ tokio = ["dep:tokio", "async"]
 doc_cfg = []
 
 [dependencies]
+winapi = { version = "0.3.9", features = ["winnt"] }
```

Comment:
`winapi` was swapped out for `windows-sys` just recently. What is the purpose of pulling it back in?

Reply:
really because in windows_sys I didn't find
#[cfg(target_pointer_width = "64")]
pub const SECURITY_DESCRIPTOR_MIN_LENGTH: usize = 40;
#[cfg(target_pointer_width = "32")]
pub const SECURITY_DESCRIPTOR_MIN_LENGTH: usize = 20;
However I will declare it within security_desciptor.rs and remove the dependency